ecocodejava.properties

```diff
@@ -1,7 +1,7 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for Java language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
 homepageUrl=https://github.com/green-code-initiative/ecoCode
-publicVersions=1.2.1
+publicVersions=1.2.1,1.3.0
```

Comment:
You can't have 2 public versions with overlapping compatibility. In this case, both versions have identical compatibility. 
Take  a look at the README. 

You should archive v1.2.1 (`archivedVersions`) and update its compatibility range to end with the current/latest SonarQube version. In this case, that would be `10.1`